Describe how to avoid dependency confusion in "secure installs" topic (#11810)
pypa/pip: docs/html/topics/secure-installs.md

```{pip-cli}
$ pip install --no-deps -e .
```

## Avoiding dependency confusion

> When dealling with private packages on your project you need to be careful to [dependency confusion](https://azure.microsoft.com/en-us/resources/3-ways-to-mitigate-risk-using-private-package-feeds/), where an attacker can claim the package on the public repository in a way that will ensure it gets chosen over the private package.

Reviewer: ```suggestion
When dealing with private packages on your project you need to be careful to [dependency confusion](https://azure.microsoft.com/en-us/resources/3-ways-to-mitigate-risk-using-private-package-feeds/), where an attacker can claim the package on the public repository in a way that will ensure it gets chosen over the private package.
```

> When dealing with private packages on your project you need to be careful to [dependency confusion](https://azure.microsoft.com/mediahandler/files/resourcefiles/3-ways-to-mitigate-risk-using-private-package-feeds/3%20Ways%20to%20Mitigate%20Risk%20When%20Using%20Private%20Package%20Feeds%20-%20v1.0.pdf), where an attacker can claim the package on the public repository in a way that will ensure it gets chosen over the private package.

Reviewer 2: ```suggestion
When dealing with private packages on your project you need to be careful to avoid [dependency confusion](https://azure.microsoft.com/mediahandler/files/resourcefiles/3-ways-to-mitigate-risk-using-private-package-feeds/3%20Ways%20to%20Mitigate%20Risk%20When%20Using%20Private%20Package%20Feeds%20-%20v1.0.pdf), where an attacker can claim the package on the public repository in a way that will ensure it gets chosen over the private package.
```

> To avoid that risk you don't want to install your private packages using {any}`--extra-index-url` like below:

Reviewer: ```suggestion
To avoid that risk you shouldn't install your private packages using {any}`--extra-index-url` like below:
```
